packages/utils/src/index.js

```diff
@@ -2,6 +2,7 @@ import pLazy from 'p-lazy';
 import pReflect from 'p-reflect';
 import isPromise from 'p-is-promise';
 import semver from 'semver';
+import ExMatch from '@mysupport/expression-match';
```

Comment:
I had to fork this to resolve audit issue in lodash, nothing changes other than lodash version and cherry picking lodash functions.